providers/twitter.json

```diff
@@ -10,6 +10,14 @@
     {
       "name": "accessToken",
       "description": "User-authorized OAuth 2.0 access token"
+    },
+    {
```

Comment:
It will become unclear for the user which use-case requires which integration parameters. But I do not have solution in mind for it.

Reply:
Yeah, agreed. Start a beglog for that?